index.bs

```diff
@@ -73,47 +73,42 @@ spec: webidl
   </h2>
   <p><em>This section is non-normative.</em></p>
   <p>
-    This document's goal is to specify an API that will help developers to
-    handle permissions on the Web platform. Web APIs have different ways to
-    deal with permissions. The [[notifications]] API allows developers to
+    This document specifies an API to query and request permissions to
```

Comment:
The document also centralizes the definitions of how other specs request permission from people.